cclient/iota_client_extended_api.c

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2018 IOTA Stiftung
+ * https://github.com/iotaledger/entangled
+ *
+ * Refer to the LICENSE file for licensing information
+ */
+
+#include "iota_client_extended_api.h"
+
+static bool is_unused_address(iota_client_service_t const* const serv,
```

Comment:
There are many reasons this function can fail (`find_transactions_req_new `, `find_transactions_res_new `, `iota_client_find_transactions `...) so it probably should return `retcode_t `